grammar.js

```diff
@@ -373,6 +377,37 @@ module.exports = grammar({
       seq(repeat($._annotation), '[', ']')
     )),
 
+    switch_expression: $ => seq(
+      'switch',
+      field('condition', $.parenthesized_expression),
+      field('body', $.switch_block)
+    ),
+
+    switch_block: $ => seq(
+      '{',
+      choice(
+        repeat($.switch_block_statement_group), 
+        repeat($.switch_rule)
+      ),
+      '}'
+    ),
+
+    switch_block_statement_group: $ => choice(
+        seq($.switch_label, ':'),
+        $.statement
+    ),
```

Comment:
Yeah, I noticed that the spec  used this "statement group" terminology. Adopting it seems like a good change. But I think that if we're going to call this node a *statement group*, it should contain a group of statements, similarly to how it is structured in the spec.

For example, a switch statement like this would just have two groups: one for `case A: case B:` and all of their resulting statements, and one for `default`:

```java
switch (i) {
  case A:
  case B:
    b();
    c();
    break;

  default:
    d();
    return;
}
```

So I think we'd need to have repetitions around the labels ands the statements. Does that make sense?

Reply:
Yup; done